web/packages/core/src/ruffle-player.ts

```diff
@@ -897,12 +898,28 @@ export class RufflePlayer extends HTMLElement {
     }
 
     /**
-     * Called when entering / leaving fullscreen
+     * Called when entering / leaving fullscreen.
      */
     private fullScreenChange(): void {
         this.instance?.set_fullscreen(this.isFullscreen);
     }
 
+    /**
+     * Prompt the user to download a file.
+     *
+     * @param href The content to download.
+     * @param name The name to give the file.
+     */
+    private saveFile(href: string, name: string): void {
```

Comment:
All `saveFile()` calls are preceded by `const blobURL = URL.createObjectURL(blob);` and followed by `URL.revokeObjectURL(blobURL);`. How about pulling that logic inside `saveFile()` itself, making it accept a `blob: Blob` in place of `href: string`?